src/app/tests/suites/certification/Test_TC_CGEN_2_1.yaml

```diff
@@ -71,4 +71,8 @@ tests:
       command: "readAttribute"
       attribute: "BasicCommissioningInfo"
       response:
-          value: { FailSafeExpiryLengthSeconds: 60 }
+          value:
+              {
+                  FailSafeExpiryLengthSeconds: 60,
+                  MaxCumulativeFailsafeSeconds: 900,
```

Comment:
Please file an issue on this: devices don't have to use those values, so a certification test can't require them.

Reply:
Is there any reasonable check that could be done? Like `> 0` or something similar?